### ISSUE TITLE
Nice to have: LUT support

### ISSUE THREAD

**Issue by fdev31** [ElvinC/gyroflow#117]
Many cameras have their .cube 3D LUT files available, for simple workflows it could be convenient to integrate some color grading gyroflow since it's almost "for free" using ffmpeg.

An option to select a .cube file and then add it to the ffmpeg pipeline would be great.
As an illustration, such video filter option does the job:

`-vf lut3d=LUTS/Insta360/ONE-R-LUT/1-INCH.CUBE:interp=tetrahedral`

**Comment by seanmavley** [comment]
I think currently you have the option to write in your custom ffmpeg command, therefore you can add in the above params during export.

![image](https://user-images.githubusercontent.com/5289083/143773420-2f0704d0-f39b-43c0-a62f-6a82433cfe97.png)

**Comment by fdev31** [comment]
Indeed, I agree, but it is not so "accessible" in my opinion for the following reasons:
- not user friendly, you need some ffmpeg knowledge to use it
- it does not allow easy selection of the .cube file (while a better integration allows a dropdown with a list of built-in CLUT for the most common cameras)
- it's quite unclear which "settings above" will be cleared if this field is provided, and how to replicate the settings
   - this last problem could be mitigated if this field somehow "auto filled in" according to the current settings, but this adds another level of complexity
